github/commitstatus.go

```diff
@@ -175,9 +183,16 @@ func httpRequestDo(req *http.Request) (httpResponse, error) {
 	response.oauthInfo = fmt.Sprintf("X-Accepted-Oauth-Scopes: %v, X-Oauth-Scopes: %v",
 		resp.Header.Get("X-Accepted-Oauth-Scopes"), resp.Header.Get("X-Oauth-Scopes"))
 
-	response.rateLimitRemaining, _ = strconv.Atoi(resp.Header.Get("x-ratelimit-remaining"))
-	rateLimitReset, _ := strconv.Atoi(resp.Header.Get("x-ratelimit-reset"))
+	response.rateLimitRemaining, _ = strconv.Atoi(resp.Header.Get("X-RateLimit-Remaining"))
+	rateLimitReset, _ := strconv.Atoi(resp.Header.Get("X-RateLimit-Reset"))
```

Comment:
same comment should apply here also: https://github.com/Pix4D/fly_helper/pull/158/files#r1201798389

> [marco-m-pix4d](https://github.com/marco-m-pix4d)(Marco Molteni) [2 minutes ago](https://github.com/Pix4D/fly_helper/pull/158/files#r1201798389)
In Go, never skip handling errors, unless there is a very specific reason.
This is equivalent to intercepting all exceptions in Python and dropping them.

Reply:
This is not an excuse in any case, but I "learned" the best practice from:
https://github.com/google/go-github/blob/f96b64ece988554ba6444cb305e6db1132a61454/github/github.go#L674

Reply:
From the documentation:

> The errors that ParseInt returns have concrete type *NumError and include err.Num = s. If s is empty or contains invalid digits, err.Err = ErrSyntax and the returned value is 0; if the value corresponding to s cannot be represented by a signed integer of the given size, err.Err = ErrRange and the returned value is the maximum magnitude integer of the appropriate bitSize and sign. 

So in case of error, the outcome of strconv.Atoi is `0`. They might have decided that `0` is OK, or they might have made a wrong assumption.

Reply:
All errors are now properly handled.